### ISSUE TITLE
[Adaptive Cards-ExpenseReport.json]: 'Expense Details' text present on the 'ExpenseReport.json' card is visually appearing as heading but not defined as heading programmatically.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9224]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**
1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Cards Visualizer application.
3. Navigate to the 'ExpenseReport.json' control present on left navigation using tab key and activate it using enter key.
4. 'ExpenseReport.json' adaptive card will appear on right sight of the pane.
5. Navigate to the 'Expense Details' text in heading mode by pressing 'H' key.
6. Verify that, 'Expense Details' text is defined inside any heading tag or not. 

**Actual Result:**
'Expense Details' text present on the 'ExpenseReport.json' card is visually appearing as heading but not defined as heading programmatically.

**Expected Result:**
'Expense Details' text present on the 'ExpenseReport.json' card is visually appearing as heading and it should be defined inside the h3 heading programmatically.

**User Impact:**
The text, visually appearing as heading is not defined under heading tag impacts the screen reader users in understanding context of the section defined under that heading as the information and relationships that are implied by visual formatting is not preserved programmatically.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/2d9ab9d3-5403-4e73-bca6-2135a9558cde)

https://github.com/user-attachments/assets/95afd350-bb78-47d1-b886-6096e39f6681

### Card JSON

```JSON
N/A
```
